bigquery/query.rb

```diff
@@ -31,22 +31,28 @@ def unique_words project_id
         # [END build_service]
 
         # [START run_query]
-        sql = "SELECT TOP(corpus, 10) as title, COUNT(*) as unique_words " +
-              "FROM [publicdata:samples.shakespeare]"
         results = bigquery.query sql
         # [END run_query]
 
         # [START print_results]
-        results.each do |row|
-          puts "#{row['title']}: #{row['unique_words']}"
+        results.each_with_index do |row, index|
+          puts "--- Row #{index+1} ---"
+          for column, value in row
+            puts "#{column}: #{value}"
```

Comment:
The sample runs any query, so it just iterates and prints out the field name/value.